clang/docs/analyzer/checkers.rst

```diff
@@ -2097,6 +2097,18 @@ This checker is a part of ``core.StackAddressEscape``, but is temporarily disabl
 
 .. _alpha-core-TestAfterDivZero:
 
+alpha.core.StdVariant (C++)
+"""""""""""""""""""""""""""
+Check if the active value accessed in an ``std::variant`` instance.
```

Comment:
This sentence has no subject (you need to add e.g. "is"). Also consider writing something slightly longer and more specific, using "warn if" instead of "check if" and clarifying what does "accessed" mean.